go/vt/sqlparser/ast.go

```diff
@@ -136,8 +144,8 @@ type Insert struct {
 }
 
 func (node *Insert) Format(buf *TrackedBuffer) {
-	buf.Myprintf("insert %vinto %v%v %v%v",
-		node.Comments,
+	buf.Myprintf("insert %v%sinto %v%v %v%v",
```

Comment:
Yeah. I've padded spaces before or after for most of these fields. They're mainly used for regenerating code. If they start to gather fancier meanings, we can always revisit.